code/game/objects/structures/lattice.dm

```diff
@@ -87,13 +86,13 @@
 	smooth = SMOOTH_TRUE
 	canSmoothWith = null
 
-/obj/structure/lattice/catwalk/New()
+/obj/structure/lattice/catwalk/Initialize()
```

Comment:
Probably not needed given we don't do anything with the argument, but I'll put it in anyway.